File: src/org/baksia/rustycage/editors/RustColorConstants.java
package org.baksia.rustycage.editors;

import org.eclipse.swt.graphics.Color;
import org.eclipse.swt.graphics.RGB;
import org.eclipse.swt.widgets.Display;

public class RustColorConstants {
    public static final Color KEYWORD = new Color(Display.getCurrent(), new RGB(160, 82, 45));
    public static final Color NEW_FILE = new Color(Display.getCurrent(), new RGB(210, 105, 30));
    public static final Color COMMENT = new Color(Display.getCurrent(), new RGB(188, 143, 143));
    public static final Color STRING = new Color(Display.getCurrent(), new RGB(70, 130, 180)); //new RGB(119, 136, 153));
    public static final Color NUMBERS = new Color(Display.getCurrent(), new RGB(104, 131, 139));
    public static final Color CONTENT_ASSIST = new Color(Display.getCurrent(), new RGB(220, 220, 220));
}

File: rustycage-plugin/src/org/rustycage/editors/RustColorConstants.scala
package org.rustycage.editors

import org.eclipse.swt.graphics.{Color, RGB}
import org.eclipse.swt.widgets.Display

object RustColorConstants {
  val KEYWORD = new Color(Display.getCurrent, new RGB(160, 82, 45))
  val NEW_FILE = new Color(Display.getCurrent, new RGB(210, 105, 30))
  val COMMENT = new Color(Display.getCurrent, new RGB(188, 143, 143))
  val STRING = new Color(Display.getCurrent, new RGB(70, 130, 180)) //new RGB(119, 136, 153)
  val NUMBERS = new Color(Display.getCurrent, new RGB(104, 131, 139))
  val CONTENT_ASSIST = new Color(Display.getCurrent, new RGB(220, 220, 220))
  val POINTER = new Color(Display.getCurrent, new RGB(205, 92, 92))
  val CHAR = new Color(Display.getCurrent, new RGB(61, 89, 171))
}